packages/editor/src/client/panel/peerProfileEditPanel.tsx

```diff
@@ -99,6 +111,22 @@ export function PeerInfoEditPanel({onClose, onSave}: ImageEditPanelProps) {
             </FormGroup>
           </Form>
         </Panel>
+        <Panel header={t('peerList.panels.callToAction')}>
```

Comment:
Also here i don't think it's necessary to create a new panel for that. Just add it to the form above.